spec/spec_helper.rb

```diff
@@ -1,8 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 
-require 'coveralls'
-Coveralls.wear!('rails') do
-  add_filter("/spec/")
+if ENV["TEST_SUITE"] == "vmdb"
```

Comment:
This seems to be around the wrong way... we always want Coveralls; we want to restrict the `coverage_helper` to `vmdb`, no?